deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/StaffOnlyPermissionJobTest.java

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.deposit.fcrepo4;
+
+import static edu.unc.lib.dl.acl.util.AccessPrincipalConstants.AUTHENTICATED_PRINC;
+import static edu.unc.lib.dl.acl.util.AccessPrincipalConstants.PUBLIC_PRINC;
+import static edu.unc.lib.dl.acl.util.UserRole.none;
+import static edu.unc.lib.dl.test.TestHelpers.setField;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.when;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+
+import edu.unc.lib.dl.persist.api.storage.StorageLocationManager;
+import edu.unc.lib.dl.persist.services.storage.StorageLocationTestHelper;
+import edu.unc.lib.dl.rdf.Cdr;
+import edu.unc.lib.dl.rdf.CdrDeposit;
+import org.apache.commons.io.FileUtils;
+import org.apache.jena.rdf.model.Bag;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.vocabulary.RDF;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.unc.lib.dl.util.RedisWorkerConstants.DepositField;
+import edu.unc.lib.dl.fedora.PID;
+
+/**
+ * @author lfarrell
+ */
+public class StaffOnlyPermissionJobTest extends AbstractDepositJobTest {
+    private final static String LOC1_ID = "loc1";
+    private final static String LOC2_ID = "loc2";
+    private final static String FILE_CONTENT1 = "Some content";
+    private final static String SOURCE_ID = "source1";
+
+    private StaffOnlyPermissionJob job;
+    private Model model;
+    private StorageLocationManager locationManager;
+    private Bag depBag;
+    private StorageLocationTestHelper locTestHelper;
```

Comment:
you don't need any of the storage location related stuff in this test since the job doesn't interact with the binaries. So you can remove all the variables below this line, and the constants that go with them.